doc/Makefile

```diff
@@ -58,6 +58,8 @@ run: install
 # Doesn't depend on $(BUILDDIR) to rebuild properly at every run.
 html: install
 	. $(VENV); $(SPHINXBUILD) -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" -w .sphinx/warnings.txt $(SPHINXOPTS)
+	@echo "\nClipping 'How to' from Table of Contents:"
+	sed -i '/toctree-l2.*>How to / s/How to \(.\)/\U\1/' _build/**/*.html
```

Comment:
thought: are the repetitive "How to"s really required in the titles? I mean, the section itself is called "How-to guides"... if we could remove them this hack wouldn't be necessary. Anyway, apparently there is no easy way to adjust the title in the sidebar and this string replacement seems to work just fine.

Reply:
> I mean, the section itself is called "How-to guides"... if we could remove them this hack wouldn't be necessary.

True, but to keep it copnsistent with other Diataxis docs, this is how we're advised to do it: have "How to" in the heading but not in navigation.

> apparently there is no easy way to adjust the title in the sidebar and this string replacement seems to work just fine

I'm not happy with this hack. It's brittle and asks for trouble down the road. But modifying the navig. titles only is not possible with Markdown sources :(